tfhe-zk-pok/src/proofs/rlwe.rs

```diff
@@ -81,9 +81,14 @@ pub struct Proof<G: Curve> {
     c_hat: G::G2,
     c_y: G::G1,
     pi: G::G1,
-    c_hat_t: Option<G::G2>,
-    c_h: Option<G::G1>,
-    pi_kzg: Option<G::G1>,
+    optional_fields: Option<ProofOptionalFields<G>>,
+}
+
+#[derive(Clone, Debug)]
+struct ProofOptionalFields<G: Curve> {
```

Comment:
Any better name for this struct and the associated field?

Reply:
done !